src/main/java/com/hazelcast/aws/security/Aws4RequestSignerReference.java

```diff
@@ -0,0 +1,120 @@
+package com.hazelcast.aws.security;
+
+import com.hazelcast.aws.AwsConfig;
+import com.hazelcast.aws.impl.Constants;
+import com.hazelcast.aws.utility.Aws4RequestSignerUtils;
+import uk.co.lucasweb.aws.v4.signer.Header;
+import uk.co.lucasweb.aws.v4.signer.HttpRequest;
+import uk.co.lucasweb.aws.v4.signer.Signer;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.hazelcast.aws.utility.StringUtil.isNotEmpty;
+
+/**
+ *
+ */
+public class Aws4RequestSignerReference implements Aws4RequestSigner {
```

Comment:
Where is this class used?

Reply:
It's part of the leftover, will be removed